help/desktop-app-install-guide.md

```diff
@@ -119,26 +120,60 @@ command:
 
 Get a peek at new features before they're released!
 
-#### macOS, Windows, and most Linux distros
+{start_tabs}
+
+{tab|most-systems}
+
+{!app-will-update-tip.md!}
+
+1. Go to the [Zulip releases][release-list] page on GitHub, and find the latest
+   version tagged with the “Pre-release” label.
+
+1. If there's a **Pre-release** that's more recent than the [latest release][latest],
+   download the appropriate Zulip beta installer or app for your system.
+
+1. To install and run Zulip, refer to the instructions for your operating
+   system in the [Install the latest release](#install-the-latest-release)
+   section above.
+
+{tab|linux-with-apt}
+
+!!! tip ""
+
+    The app will be updated automatically to future versions when you do a
+    regular software update on your system, e.g., with
+    `sudo apt update && sudo apt upgrade`.
+
+#### You don't have the Zulip app installed
+
+1. Enter the following commands into a terminal:
 
-Start by finding the latest version marked "Pre-release" on the
-[release list page][release-list].  There may or may not be a "Pre-release"
-later than the latest release. If there is, download the appropriate Zulip
-installer or app from there, and follow the instructions for your operating
-system above.
+        sudo curl -fL -o /etc/apt/trusted.gpg.d/zulip-desktop.asc \
+            https://download.zulip.com/desktop/apt/zulip-desktop.asc
+        echo "deb https://download.zulip.com/desktop/apt beta main" | \
+            sudo tee /etc/apt/sources.list.d/zulip-desktop.list
+        sudo apt update
+        sudo apt install zulip
```

Comment:
Can you use a fenced code block here? 

Same question in the other entry in the file, they're just easier to work with and probably should be part of our /help/ style guide.